src/main/java/net/glowstone/util/Wearable.java

```diff
@@ -0,0 +1,120 @@
+package net.glowstone.util;
+
+import org.bukkit.enchantments.EnchantmentTarget;
+import org.bukkit.Material;
+import java.util.Map;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.inventory.ItemStack;
+import java.util.HashMap;
+import java.util.Set;
+
+public final class Wearable {
+    
+    private static Map<Material, WearableReturn> map;
+    private static boolean intalized = false;
+    
+    private class WearableReturn {
+        private EnchantmentTarget wearablePosition;
+        private int armorPoints;
+        
+        public WearableReturn(EnchantmentTarget position, int points) {
+            this.wearablePosition = position;
+            this.armorPoints = points;
+        }
+        public EnchantmentTarget wearPosition() {
+            return wearablePosition;
+        }
+        public int armorPoints() {
+            return armorPoints;
+        }
+    }
+    
+    protected Map<Material, WearableReturn> createMap() {
+        Map<Material, WearableReturn> map2 = new HashMap<>();
```

Comment:
You can assign `map` to `new HashMap<>()` in the field declaration. It doesn't need to be separated.

Then, you can directly add entries to `map` instead of using `map2` and then assigning `map` to `map2`.